### PR TITLE
[card-146] fetch branch/author from environment

### PR DESCRIPTION
**Pull request by iffyuva**

### CODE REVIEW

File: lib/integration_diff/runner.rb [hunk 1]
```diff
@@ -1,11 +1,12 @@
 require 'time'
 require 'json'
+require 'integration_diff/run_details'
 
 module IntegrationDiff
   class Runner
     include Capybara::DSL
 
-    DIR = 'tmp/idff_images'
+    DIR = 'tmp/idiff_images'
```
Comment (goromlagche) [inline lib/integration_diff/runner.rb]: should we use `freeze` here?

Comment (iffyuva) [inline lib/integration_diff/runner.rb]: agreed, will do.

Comment (iffyuva) [inline lib/integration_diff/runner.rb]: done 👍

File: lib/integration_diff/runner.rb [hunk 2]
```diff
@@ -54,8 +57,9 @@ def draft_run
 
       # will have to make it configurable. ie, read from env.
       # https://github.com/code-mancers/integration-diff.rb/pull/4#discussion-diff-42290464
```
Comment (goromlagche) [inline lib/integration_diff/runner.rb]: we can remove these comments I guess.

Comment (iffyuva) [inline lib/integration_diff/runner.rb]: done, please check.

**Pull request merged**